### PR TITLE
Expand VAMC tests

### PR DESCRIPTION
**Pull request by mpelzsherman**
## Description
Add more VAMC tests per https://github.com/department-of-veterans-affairs/va.gov-team/issues/14049

Note: this PR adds tests for the main facility home page, /pittsburgh-health-care.

Once this has been merged, I'll make a second PR for the location detail pages.

### CODE REVIEW

**Comment by zacharymorel** [comment]
So the purpose here for this test is just a smoke test to see make sure the template populates with the desired content needed?

**Comment by mpelzsherman** [comment]
> So the purpose here for this test is just a smoke test to see make sure the template populates with the desired content needed?

Well to me it's more than "just a smoke test". This is mission-critical and if the content is missing, something is majorly wrong and we should not merge the changes that cause the content to disappear.

This is still WIP - sorry I accidentally created it in non-draft mode initially.

File: package.json
Comment (mpelzsherman) [inline package.json:11]: This won't be needed if we merge #15601.

File: src/site/includes/debug.drupal.liquid
Comment (mpelzsherman) [inline src/site/includes/debug.drupal.liquid:3]: Adds JSON data to a global variable so we can access in Cypress tests.

**Review by LinnJS — APPROVED** [review]
Approving these changes since our teams review popped up here not sure why it did as none of our dirs were touched in this work.

The only dirs we are responsible for are `applications/hca` & `applications/caregivers` maybe it was an error when I added HCA to our ownership?

**Review by brangi — APPROVED** [review]

File: package.json
Comment (theodur) [reply package.json:11]: I haven't seen recent PR's com across this issue. It also looks like your [pipeline](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/expand_vamc_tests/8/pipeline) is still failing. Have you tried re running again?

Comment (mpelzsherman) [reply package.json:11]: That's probably because the CI environment uses a cache of the drupal data.

When that cache is updated, I expect all of the PR's will start failing.

Comment (theodur) [reply package.json:11]: You'll probably need to increase the `max-old-space-size` in `jenkins/common.groovy` as well.

Comment (mpelzsherman) [reply package.json:11]: Looks that way, yes. Hopefully once #15601 is merged we can set it back if need be.

**Review by zacharymorel — APPROVED** [review]
LGTM

File: src/site/includes/debug.drupal.liquid
Comment (kelsonic) [reply src/site/includes/debug.drupal.liquid:3]: @ncksllvn this is the only change we need to look at I think, lgtm but wanted to get your 👀 🙂

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉 Please wait for @ncksllvn's feedback though in case he sees anything with `window.contentData`, we may already have a global variable for the content data I'm just unaware of

File: src/site/includes/debug.drupal.liquid
Comment (ncksllvn) [reply src/site/includes/debug.drupal.liquid:3]: To save a little on the file size can we change the next line to ->

```js
console.info('Metalsmith template generated from content:', window.contentData);
```

**Review by ncksllvn — APPROVED** [review]
I left a suggestion to keep the content data from being outputted to the HTML twice but otherwise LGTM

**Review by theodur — APPROVED** [review]

File: src/site/includes/debug.drupal.liquid
Comment (mpelzsherman) [reply src/site/includes/debug.drupal.liquid:3]: Thanks - I'll change this in my 2nd PR!

**Pull request merged**
